request-management-api/request_api/models/FOIMinistryRequests.py

```diff
@@ -354,6 +354,18 @@ def getrequestssubquery(cls, groups, filterfields, keyword, additionalfilter, us
         #subquery for getting extension count
         subquery_extension_count = _session.query(FOIRequestExtension.foiministryrequest_id, func.count(distinct(FOIRequestExtension.foirequestextensionid)).filter(FOIRequestExtension.isactive == True).label('extensions')).group_by(FOIRequestExtension.foiministryrequest_id).subquery()
 
+        #subquery for getting all current oipcs for foiministry request that have outcome as none (undecided outcomes)
+        subquery_oipc_nooutcomes_sql = """
+        SELECT distinct on (foiministryrequest_id) foiministryrequest_id, foiministryrequestversion_id, outcomeid 
```

Comment:
This query will return requests with outcomeid != null as well as all the null ones. You might want to add "WHERE outcomeid is null" to this query. Or since you're filtering for them later anyways, you can also change the variable name to something like "subquery_with_oipc". mostly about consistency so it doesn't get confusing later on, incase somebody is reusing this subquery

Reply:
will adjust naming convetion for the variable!